docs/bridge/docs/03-Developers/REST-API/index.md

```
@@ -0,0 +1,279 @@
+#### Parameters
+
+* `chain` (int): `chainId` of the desired chain.
+* `fromToken` (string): Address of token to swap from.
+* `toToken` (string): Address of token to swap to.
+* `amount` (int): Amount of `fromToken` to swap.
+
+#### Returns
+
+* `routerAddress` (string): The address of the router contract
+* `maxAmountOut {` (object): The maximum amount of tokens that can be swapped out.
+    * `type:` (string): The data type
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)